src/routes/Document/components/AddAndUpdateApiDoc.js

```diff
@@ -277,9 +282,16 @@ class AddAndUpdateApiDoc extends Component {
               ],
               initialValue: state
             })(
-              <Input
-                placeholder={getIntlContent("SHENYU.DOCUMENT.APIDOC.STATE")}
-              />
+              <Radio.Group buttonStyle="solid">
+                {Object.values(STATE_TYPE).map((e, i) => {
```

Comment:
STATE_TYPE.map((e, i) => ...

Reply:
I refer to the traversal method of other arrays in this file, so I am consistent with him

Reply:
I changed the way the array is traversed